src/csvcubed/utils/version.py

```diff
@@ -7,6 +7,6 @@
 from csvcubed.__init__ import __version__
 
 
-def _get_csvcubed_version():
+def get_csvcubed_version():
```

Comment:
Also, maybe call it `get_csvcubed_version_uri` to avoid ambiguity?

Reply:
Changed function name